src/apps/chatter/reducers/room-passwords.js

```diff
@@ -0,0 +1,15 @@
+export const roomPasswords = (action, state) => {
+  switch (action.type) {
+    case 'SET_ROOM_PASSWORD':
```

Comment:
Should declare constants so they can be exported: https://github.com/inputlogic/elements/blob/master/components/use-request/reducer.js#L11